app/views/api_docs/reference/v0.1/_reference.md

```diff
@@ -2184,19 +2176,670 @@ Withdraw a trainee.
 </details>
 
 <details class="govuk-details">
-  <summary class="govuk-details__summary"><code>HTTP 422</code><span> - Unprocessable Entity</span></summary>
+  <summary class="govuk-details__summary">HTTP 404<span> - Not found</span></summary>
   <div class="govuk-details__text">
     <pre class="json-code-sample">
     {
       "errors": [
         {
-          "error": "UnprocessableEntity",
-          "message": "Withdraw date Choose a withdrawal date"
+          "error": "NotFound",
+          "message": "Degree(s) not found"
         }
       ]
     }
     </pre>
   </div>
 </details>
 
+<details class="govuk-details">
+  <summary class="govuk-details__summary">HTTP 401<span> - Unauthorized</span></summary>
+  <div class="govuk-details__text">
+    <pre class="json-code-sample">
+    {
+      "error": "Unauthorized"
+    }
+    </pre>
+  </div>
+</details>
+
 ---
+
+
+## Objects
+
+### Trainee object
+
+<dl class="govuk-summary-list">
+  <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">
+    <dt class="govuk-summary-list__key"><code>trainee_id</code></dt>
+    <dd class="govuk-summary-list__value">
+      <p class="govuk-body">
+        string (limited to 24 characters)
+      </p>
+      <p class="govuk-body">
+        The unique ID of the trainee in the Register system. Used to identify the trainee when using <a href="/api-docs/reference#contents">endpoints</a> which require a <code>trainee_id</code>.
+      </p>
+      <p class="govuk-body">
+        Example: <code>37T2Vm9aipqSVokbhWUMjedu</code>
+      </p>
+    </dd>
+  </div>
+  <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">
+    <dt class="govuk-summary-list__key"><code>provider_trainee_id</code></dt>
+    <dd class="govuk-summary-list__value">
+      <p class="govuk-body">
+        string
+      </p>
+      <p class="govuk-body">
+        The unique ID of the trainee in the Provider's student record system (SRS). Coded according to the <a href="https://www.hesa.ac.uk/collection/c23053/e/ownstu">HESA provider's own identifier for student field</a>
+      </p>
+      <p class="govuk-body">
+        Example: <code>99157234</code>
+      </p>
+    </dd>
+  </div>
+  <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">
+    <dt class="govuk-summary-list__key"><code>apply_id</code></dt>
```

Comment:
Should this be `apply_application_id` instead based on [trainee_serializer/v1_spec.rb:14](https://github.com/DFE-Digital/register-trainee-teachers/blob/6904-update-api-docs-with-request-object-details/spec/serializers/trainee_serializer/v01_spec.rb#L14)?

Reply:
Hmm, good question. I think the `apply_application_id` is the foreign key for linking Trainees to Apply Applications, so that's slightly different to this.
I think this is partly an internal/external terminology thing where we wanted to keep it simply `apply_id` externally but internally this is the `application_choice_id`. However, I think this is also different to the `application_choice_id` on Trainee so we'll need to look into this further.
For now I'm changing it to `application_id` to be consistent with the [Apply API](https://www.apply-for-teacher-training.service.gov.uk/api-docs/v1.4/reference#application-object).